__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => {
+  db.end();
+});
+
+describe.only("/api/categories", () => {
+  test("GET - 200: responds with an array of category objects with slug and description as properties", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(4);
+        body.forEach((category) => {
+          expect(category).toEqual({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("GET - 404: responds with an error message, (Not found!), when passed an invalid endpoint", () => {
```

Comment:
Test does not apply for /api/categories. Please can it have its own describe block